app/backend/lib/reporting/reporting.ts

```diff
@@ -0,0 +1,58 @@
+import { Router } from 'express';
+import RateLimit from 'express-rate-limit';
+import {
+  compareAndGenerateGcpeReport,
+  compareGcpeReports,
+  generateGcpeReport,
+  regenerateGcpeReport,
+} from './gcpe';
+
+const reporting = Router();
+
+const limiter = RateLimit({
+  windowMs: 1 * 60 * 1000,
+  max: 2000,
+});
+
+reporting.get('/api/reporting/gcpe', limiter, async (req, res) => {
```

Comment:
I notice for these we're not using the same authentication check we're using for other api routes. Is the expectation here for the DB to deny the request here and let it fail vs a 404 (as we're doing now) or a 403

Reply:
I was not attempting to restrict it and rather refine it later, but yes the call will fail if you don't have proper auth and will result in an error.